### ISSUE TITLE
Add docs about ST7789v2 display

### ISSUE THREAD

**Issue by iBobik** [esphome/feature-requests#2578]
**Describe the problem you have/What new integration you would like**

I do not know how to configure platform and pins.

**Please describe your use case for this integration and alternatives you've tried:**

I want to use M5Stack Station's display, but it is not clear how to configure it:
- ST7789v2 is not in supported list, but maybe M5STACK or ST7789V is compatible
- `dc_pin` is required to be configured, but there is no such pin in Station's docs

**Additional context**

https://docs.m5stack.com/en/core/station_485
![Snímek obrazovky 2024-01-25 v 19 39 27](https://github.com/esphome/feature-requests/assets/614232/ec0aeb8b-665d-4dd8-8844-9b3dcd443556)

![Snímek obrazovky 2024-01-25 v 19 29 54](https://github.com/esphome/feature-requests/assets/614232/67c648a4-141f-46ed-8395-5f6118ca6570)

**Comment by tiaanv** [comment]
Did you get any feedback on this?  Is it supported?

**Comment by nvtkaszpir** [comment]
yes, see https://github.com/esphome/issues/issues/3591#issuecomment-2569732913
